src/canvasObject.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var canvasObject= function(canvas) {
```

Comment:
I imagine eventually we'll want this to be a robust object with methods of its own. For that I recommend something like this: 

```
'use strict';

var CanvasObject = function(config) {
    this.needed = false;
    this.visible = true;
    this.clipRegion = null;
    this.opacity = 1;
    this.position = {
        x: 0,
        y: 0
    };
    this.placeholder = '/some-image.jpg';
    this.tileSourceUrl = config.canvas.images[0].resource.service['@id'] + '/info.json';
};

CanvasObject.prototype = {
   myMethod: function() {
   } 
};

module.exports = CanvasObject;
```

...which would be created in main.js like so: 

```
 canvasObjects[canvas['@id']] = CanvasObject({
   canvas: canvas
 });
```